cypress/e2e/shared/dashboardsView.test.ts

```diff
@@ -287,224 +289,371 @@ describe('Dashboard', () => {
-              // save cell
-              cy.getByTestID('save-cell--button').click()
+                cy.getByTestID(
+                  `variable-dropdown-input-typeAhead--${bucketVarName}`
+                ).type('an')
 
-              // selected value in cell context is 1st value
-              cy.window()
-                .pipe(getSelectedVariable(dashboard.id, 0))
-                .should('equal', bucketOne)
+                // dropdown should  be showing: anotherBucket', 'randomBucket'
 
-              // selected value in dashboard is 1st value
-              cy.getByTestID(`variable-dropdown--${bucketVarName}`).should(
-                'contain',
-                bucketOne
-              )
-              cy.window()
-                .pipe(getSelectedVariable(dashboard.id, 0))
-                .should('equal', bucketOne)
+                // hit down arrow once:
 
-              // TESTING MAP VARIABLE
-              // selected value in dashboard is 1st value
-              cy.getByTestID(`variable-dropdown--${mapTypeVarName}`).should(
-                'contain',
-                'k1'
-              )
-              cy.window()
-                .pipe(getSelectedVariable(dashboard.id, 2))
-                .should('equal', 'v1')
+                cy.getByTestID(
+                  `variable-dropdown-input-typeAhead--${bucketVarName}`
+                ).type('{downarrow}')
 
-              // select 2nd value in dashboard
-              cy.getByTestID('variable-dropdown--button')
-                .eq(1)
-                .click()
-              cy.get(`#k2`).click()
+                // first element should be active (highlighted)
+                cy.get(`#anotherBucket`).should('have.class', 'active')
 
-              // selected value in dashboard is 2nd value
-              cy.getByTestID(`variable-dropdown--${mapTypeVarName}`).should(
-                'contain',
-                'k2'
-              )
-              cy.window()
-                .pipe(getSelectedVariable(dashboard.id, 2))
-                .should('equal', 'v2')
+                // hit down arrow again
+                cy.getByTestID(
+                  `variable-dropdown-input-typeAhead--${bucketVarName}`
+                ).type('{downarrow}')
 
-              // open VEO
-              cy.getByTestID('cell-context--toggle').click()
-              cy.getByTestID('cell-context--configure').click()
-              cy.getByTestID('toolbar-tab').should('be.visible')
+                // next one should be active (first should NOT be active)
+                cy.get(`#anotherBucket`).should('not.have.class', 'active')
+                cy.get(`#randomBucket`).should('have.class', 'active')
 
-              // selected value in cell context is 2nd value
-              cy.window()
-                .pipe(getSelectedVariable(dashboard.id, 2))
-                .should('equal', 'v2')
+                // now; press return/enter to set it:
+                cy.getByTestID(
```

Comment:
is the spacing off here?

Reply:
pretty-quick did this ...will adjust but if pretty-quick changes it back; well than it's a prettier issue